static/app/utils/discover/fieldRenderers.tsx

```diff
@@ -296,11 +301,47 @@ type SpecialFields = {
   'user.display': SpecialField;
 };
 
+const DownloadCount = styled('span')`
+  padding-left: 6px;
+`;
+
 /**
  * "Special fields" either do not map 1:1 to an single column in the event database,
  * or they require custom UI formatting that can't be handled by the datatype formatters.
  */
 const SPECIAL_FIELDS: SpecialFields = {
+  // This is a custom renderer for a field outside discover
+  // TODO - refactor code and remove from this file or add ability to query for attachments in
+  attachments: {
+    sortField: null,
+    renderFunc: data => {
+      const attachments: Array<IssueAttachment & {url: string}> = data.attachments;
```

Comment:
This typing is a little awkward, but fine. I think you could save a bunch of work by passing the organization and project into the renderer (which looks like a common thing to do!) and concatenating the URLs here instead of in the parent component

Reply:
Yeah, gonna address that in a code cleanup PR

Reply:
Resolved in https://github.com/getsentry/sentry/pull/40147/files